### DISCUSSION TITLE
Does Koboldcpp allow ctx values between 16384 and 24576?

### DISCUSSION THREAD

**Discussion by ReMeDy-TV** [LostRuins/koboldcpp#828] [Q&A]
I'm noticing the ctx is on a fixed slider, and offers 16384 and 24576 but is there an in-between option? I know via Runpod I can run this flag:
--contextsize x

Where x equals the desired context, but can it be an in-between value?

**Comment by LostRuins** [comment]
Yes you can set any context value you want. The slider is just for convenience. --contextsize 20000 is perfectly fine
